hazelcast-openshift-rhel/Dockerfile

```diff
@@ -0,0 +1,93 @@
+FROM registry.access.redhat.com/rhel7
+MAINTAINER Hazelcast, Inc. Integration Team <info@hazelcast.com>
+
+ENV HZ_HOME /opt/hazelcast/
+ENV HZ_CP_MOUNT ${HZ_HOME}/external
+ENV JAVA_HOME /usr/lib/jvm/java
+ENV LANG en_US.utf8
+
+ENV USER_NAME=hazelcast
+ENV USER_UID=10001
+
+ARG HZ_VERSION=3.8-RC1
+ARG HZ_KUBE_VERSION=0.9.3
```

Comment:
This should be updated as v1.0.0 is released. (https://github.com/hazelcast/hazelcast-kubernetes/releases/tag/v1.0.0)